tidb-cloud/integrate-tidbcloud-with-airbyte.md

```diff
@@ -0,0 +1,94 @@
+
+![img](/media/tidb-cloud/integration-airbyte-ready.jpg)
+
+## Set up the TiDB connector
+
+Conveniently, the steps are the same for setting TiDB as the source and the destination.
+
+1. Click **Sources** or **Destinations** in the sidebar and choose TiDB type to create a new TiDB connector.
+
+2. Fill in the following parameters:
+
+    - Host: The host domain of TiDB
+    - Port: The port of the database
+    - Database: The database that you want to sync the data
+    - Username: The username to access the database
+    - Password: The password of the username
+
+3. Enable **SSL Connection**, and set TLS protocols to **TLSv1.2** or **TLSv1.3** in **JDBC URL Params**.
+
+    > Note:
+    >
+    > - TiDB Cloud supports TLS connection. You can choose your TLS protocols in **TLSv1.2** and **TLSv1.3**, for example, `enabledTLSProtocols=TLSv1.2`.
+    > - If you want to disable TLS connection to TiDB Cloud via JDBC, you need to set useSSL to `false` in JDBC URL Params specifically and close SSL connection, for example, `useSSL=false`.
+    > - TiDB Serverless Tier only supports TLS connections.
+
+4. Click **Set up source** or **destination** to complete creating the connector. The following screenshot shows the configuration of TiDB as the source.
+
+![img](/media/tidb-cloud/integration-airbyte-parameters.jpg)
+
+For more details about the TiDB connector, see [TiDB Source](https://docs.airbyte.com/integrations/sources/tidb) and [TiDB Destination](https://docs.airbyte.com/integrations/destinations/tidb).
+
+## Set up the connection
+
+After setting up the source and destination, you can build and configure the connection. You can use any combination of sources and destinations, such as TiDB to Snowflake, and CSV file to TiDB.
+
+The following steps use TiDB as both source and destination. Other connectors may have different parameters.
+
+1. Click **Connections** in the sidebar and then click **New Connection**.
+2. Select the previously established source and destination.
+3. Go to the **Set up** connection panel and create a name for the connection, such as "${source_name} - ${destination-name}".
+4. Set **Replication frequency** to **Every 24 hours** which means the connection replicates data once a day.
+5. Set **Destination Namespace** to **Custom format** and set **Namespace Custom Format** to **test** to store all data in the `test` database.
+6. Choose the **Sync schema** to **Full refresh | Overwrite**.
+
+    > **Tips**
+    >
+    > The TiDB connector supports both Incremental and Full Refresh syncs. In Incremental mode, Airbyte only reads records added to the source since the last sync job. The first sync using Incremental mode is equivalent to Full Refresh mode. In Full Refresh mode, Airbyte reads all records in the source and replicates to the destination in every sync task. You can set the sync mode for every table named **Namespace** in Airbyte individually.
```

Comment:
```suggestion
    > The TiDB connector supports both Incremental and Full Refresh syncs.
    >
    > - In Incremental mode, Airbyte only reads records added to the source since the last sync job. The first sync using Incremental mode is equivalent to Full Refresh mode.
    > - In Full Refresh mode, Airbyte reads all records in the source and replicates to the destination in every sync task. You can set the sync mode for every table named **Namespace** in Airbyte individually.
```